src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    public boolean isDone;
+    public String name;
+
+    // constructor
+    public Task(String name) {
+        this.name = name;
+        this.isDone = false;
+    }
+
+    public Task setDone() {
+        Task doneTask = new Task(this.name);
+        doneTask.isDone = true;
+        return doneTask;
+    }
+
+    public String getStatusIcon() {
+        return (this.isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Just a suggestion, but maybe a good practice would be to make tick and X symbols constants. for e.g. `TICK_SYMBOL = "\u2713"` and `CROSS_SYMBOL = "\u2718"` to avoid having magic strings in the code.